pyogrio/_err.pyx

```diff
@@ -1,11 +1,25 @@
-# ported from fiona::_err.pyx
-from enum import IntEnum
+"""Error handling code for GDAL/OGR.
+
+Ported from fiona::_err.pyx
+"""
+
+import contextlib
+import logging
 import warnings
+from contextvars import ContextVar
+from enum import IntEnum
+from itertools import zip_longest
 
 from pyogrio._ogr cimport (
     CE_None, CE_Debug, CE_Warning, CE_Failure, CE_Fatal, CPLErrorReset,
     CPLGetLastErrorType, CPLGetLastErrorNo, CPLGetLastErrorMsg, OGRErr,
-    CPLErr, CPLErrorHandler, CPLDefaultErrorHandler, CPLPushErrorHandler)
+    CPLErr, CPLErrorHandler, CPLDefaultErrorHandler, CPLPopErrorHandler,
+    CPLPushErrorHandler, CPLPushErrorHandlerEx)
+
+log = logging.getLogger(__name__)
```

Comment:
Do we need to logging part of this? (in contrast to fiona, we in general don't enable logging)

Reply:
Not sure how interesting the logging is in general, but as it was apparently consciously removed when porting from fiona I suppose it's not that interesting, so I'll remove it.